internal/featureflags/permanent.go

```diff
@@ -97,3 +97,11 @@ func BuildSimpleClassicURL() FeatureFlag {
 		defaultEnabled: true,
 	}
 }
+
+// LogFile returns the feature flag to control whether log files shall be created or not
+func LogFile() FeatureFlag {
+	return FeatureFlag{
+		envName:        "MONACO_LOG_FILE",
```

Comment:
```suggestion
		envName:        "MONACO_LOG_FILE_ENABLED",
```

Reply:
I think a name that implies a boolean rather than the file name would be more appropriate